Advanced_Projects/PyFlowViz/pyflowviz.py

```
@@ -0,0 +1,254 @@
+#!/usr/bin/env python3
+"""
+PyFlowViz: Code-to-Flowchart Auto Generator
+CLI entry point
+"""
+
+import os
+import sys
+import argparse
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)